src/peft/utils/loftq_utils.py

```diff
@@ -227,3 +233,139 @@ def loftq_init(weight: Union[torch.Tensor, torch.nn.Parameter], num_bits: int, r
     lora_A, lora_B = R, L
 
     return dequantized_weight.to(device=device, dtype=dtype), lora_A, lora_B
+
+
+def _check_model_path_loftq(model_path, peft_model):
+    """Check if there is a local model file corresponding to the given model, then format it properly."""
+    if model_path is None:
+        try:
+            model_path = snapshot_download(peft_model.base_model.config._name_or_path, local_files_only=True)
+        except AttributeError as exc:
+            raise ValueError(
+                "The provided model does not appear to be a transformers model. In this case, you must pass the "
+                "model_path to the safetensors file."
+            ) from exc
+        except LocalEntryNotFoundError as exc:
+            raise ValueError("The model.safetensors file must be present on disk, but it could not be found.") from exc
```

Comment:
```suggestion
            raise ValueError("The adapter_model.safetensors / adapter_model.bin file must be present on disk, but it could not be found.") from exc
```
Assuming you're looking for the adapter model file?